### PR TITLE
Make "server_name" configurable in "logging-sentry" extension

### PR DESCRIPTION
**Pull request by medvedev**

### CODE REVIEW

File: extensions/logging-sentry/runtime/src/main/java/io/quarkus/logging/sentry/SentryConfig.java [hunk 1]
```diff
@@ -76,4 +76,12 @@
      */
     @ConfigItem
     public Optional<String> release;
+
+    /**
+     * Server name
+     *
+     * To set the server name that will be sent with each event.
```
Comment (machi1990) [inline extensions/logging-sentry/runtime/src/main/java/io/quarkus/logging/sentry/SentryConfig.java:83]:
```suggestion
     * Sets the server name that will be sent with each event.
```

Comment (medvedev) [reply extensions/logging-sentry/runtime/src/main/java/io/quarkus/logging/sentry/SentryConfig.java]: Addressed

**Comment by gsmet** [comment]
Rebased to fix CI.

**Comment by slava-med** [comment]
> Rebased to fix CI.

Didn't help :(

**Comment by gsmet** [comment]
Yeah, not related to it, we will merge anyway. I will ask Andy to have a look as he is the Sentry extension maintainer.

**Review by ia3andy — APPROVED** [review]
Good good!

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**